MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h

```diff
@@ -3,8 +3,31 @@
 File used to define the functions that will be needed
 as the project progesses.
  */
- 
+
+#include <WiFiNINA.h>
+
  void initBluetooth();
  void intiWifi();
  void connectBluetooth();
- void connectWifi();
\ No newline at end of file
+void connectWifi(){
```

Comment:
Lets put this function into a new file for now, connect wifi, like I have it for initializing the bluetooth, the code will be cleaner that way

Reply:
Okay, I understand. Did you already put my code in a new file or did you want me to do that?

Reply:
Also, what is the difference between initwifi and connectwifi?

Reply:
Ill change the file actually dont worry about it